verification/curator-service/ui/src/components/NewCaseForm.tsx

```diff
@@ -59,6 +59,7 @@ interface FormValues {
     age?: number;
     ethnicity?: string;
```

Comment:
sometimes we have this ?: string; and sometimes string | null. Can we be consistent please?

Reply:
Yea I agree it's not great. I chose either undefined or null for each based on what the defaults are in the field selector when you remove the field and what the backend accepted for "not-populated". I'll look again now and see if those can be aligned and do it as it's own PR if so.